hi-ml-cpath/src/health_cpath/utils/output_utils.py

```diff
@@ -254,24 +255,22 @@ def should_save_test_outputs(self, is_global_rank_zero: bool = True) -> bool:
 class DeepMILOutputsHandler:
     """Class that manages writing validation and test outputs for DeepMIL models."""
 
-    def __init__(self, outputs_root: Path, n_classes: int, tile_size: int, level: int,
+    def __init__(self, outputs_root: Path, n_classes: int, tile_size: int,
                  class_names: Optional[Sequence[str]], primary_val_metric: MetricsKey,
                  maximise: bool, val_plot_options: Collection[PlotOption],
-                 test_plot_options: Collection[PlotOption], wsi_has_mask: bool = True,
-                 backend: str = "cuCIM", val_set_is_dist: bool = True) -> None:
+                 test_plot_options: Collection[PlotOption], loading_params: LoadingParams = LoadingParams(),
```

Comment:
Avoid mutable default arguments. Long example why can be found here: https://towardsdatascience.com/python-pitfall-mutable-default-arguments-9385e8265422